Source/WebCore/platform/graphics/ImageBuffer.cpp

```diff
@@ -500,6 +507,15 @@ void ImageBuffer::setVolatilityState(VolatilityState volatilityState)
         backend->setVolatilityState(volatilityState);
 }
 
+void ImageBuffer::setVolatileAndPurgeForTesting()
+{
+    releaseGraphicsContext();
```

Comment:
This must be redundant?

Reply:
I was concerned that consumers might leave a transform in place, resulting in the following clearRect not affecting the right pixels and making the test silently stop testing the required behaviour.